### DISCUSSION TITLE
Optional int parameter 'checkok' is present but cannot be translated into a null value due to being declared as a primitive type. Consider declaring it as object wrapper for the corresponding primitive type.

### DISCUSSION THREAD

**Discussion by Seo-de** [FFarmFFarm/FarmFarm#202] [오류모음]
<h3>[문제 원인]</h3>
Spring에서 @RequestParam 작성 시 defaultValue를 작성해주지 않아
 required = false에 의해 int형이 0이 아닌 null값으로 들어가 오류가 뜸.

<h3>[문제 해결]</h3>
defaultValue="0"으로 작성해줘 혹시 값이 안들어갔을 경우 default값을 0으로 설정해줌.
@RequestParam(value="checkok", required = false, defaultValue="0") int checkok 로 수정해주었음.

<img width="1052" alt="오류3" src="https://user-images.githubusercontent.com/110797113/210248997-dac6217d-b489-4163-986b-9717847132b9.PNG">

**Comment by luejenie** [comment]
도은아.. 나의 은인..
